### ISSUE TITLE
Feature request: Library should provide default implementations of sorting, filtering, paging

### ISSUE THREAD

**Issue by ogix** [valor-software/ng2-table#322]
I guess most of the users (including me) need default implementation of sorting, filtering, paging which is available in the demo page. I do not want to copy-paste the same implementation in all my components that have tables. You may say that I could create my own component which wraps this component with my own implementation but it requires additional work that not everyone would like to do. 
Ideally it should provide default functionality with ability to override. 
What do you think?

**Comment by ghost** [comment]
We're running into the same issue using ng2-table across multiple projects with the same config. Perhaps we could share our implementations and create a PR for a change that does this?

**Comment by windmaomao** [comment]
yes, this is a bit interesting that the library implemented all the UI but with no functions :) Should it be documented on the front page ?
